### PR TITLE
colw/Loading state for single validator

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -2,6 +2,8 @@
   <TmPage
     :managed="true"
     :data-empty="!validator.operatorAddress"
+    :loading="$apollo.loading"
+    :loaded="!$apollo.loading"
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: that doesn't work. this needs to be a value set after the initial load. i.e. in the `result` hook of the `validator` query.

Comment (faboweb) [reply src/components/staking/PageValidator.vue]: Currently it defeats the purpose of `loaded` as it is set to `false` on ever new loading process.

Comment (colw) [reply src/components/staking/PageValidator.vue]: It appears to work fine for me. Also when the validator address is unknown.

Comment (faboweb) [reply src/components/staking/PageValidator.vue]: What happens if you delegate to the validator and there is a refetch of the rewards?

Comment (colw) [reply src/components/staking/PageValidator.vue]: I see, a new transaction in the subscription would cause it to show the loading component.

Comment (faboweb) [reply src/components/staking/PageValidator.vue]: 💯

File: src/components/staking/PageValidator.vue [hunk 2]
```diff
@@ -309,6 +312,7 @@ export default {
         }
       },
       update: result => {
+
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue]: this new line is not necessary

how is it that these types of lint things are getting through?

File: src/components/staking/PageValidator.vue [hunk 3]
```diff
@@ -325,6 +329,10 @@ export default {
           ...result.validator,
           statusDetailed: getStatusText(result.validator.statusDetailed)
         }
+      },
+      result(queryResult) {
+        console.log(queryResult)
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue]: console not needed 😄

**Comment by jbibla** [comment]
unit tests are fixed by adding
```
      if (this.network.testnet) {
        return true
      }
```
to `checkFeatureAvailable()` in `ActionModal.vue`

**Pull request merged**